cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -5593,4 +5593,15 @@
       Health Check HTTP service bind port; if 0, binds to a random port
     </description>
   </property>
+
+  <!-- Source Control Configuration  -->
```

Comment:
suggestion: `Source Control Configuration` --> `Configuration for Source Control Management`

Reply:
✅